schema/spacewalk/upgrade/susemanager-schema-4.4.0-tosusemanager-schema-4.4.1/001-remove-cluster_admin.sql

```diff
@@ -0,0 +1,4 @@
+
+DELETE FROM rhnUserGroupMembers WHERE user_group_id IN (SELECT G.id FROM rhnusergroup G, rhnusergrouptype GT WHERE G.group_type = GT.id AND GT.label = 'cluster_admin');
+DELETE FROM rhnUserGroup WHERE group_type IN (SELECT id FROM rhnusergrouptype WHERE label = 'cluster_admin');
+DELETE FROM rhnUserGroupType WHERE label = 'cluster_admin';
```

Comment:
I think you may also need to clean rows from the table "rhnuserextgroupmapping" just in case since it is referencing the table "rhnUserGroupType"

Reply:
addressed